.codecov.yml

```diff
@@ -0,0 +1,6 @@
+# Coverage of the following flickers notoriously despite no changes being made.
+# We'll ignore them until we find a way to have reliable coverage measurements for them.
```

Comment:
we should have JIRA tasks for adding unit tests for each of these files before ignoring them